test/run_test.py

```diff
@@ -1029,14 +1029,14 @@ def handle_log_file(
     with open(file_path, "rb") as f:
         full_text = f.read().decode("utf-8", errors="ignore")
 
+    new_file = "test/test-reports/" + sanitize_file_name(
```

Comment:
So, this is not really about Windows log printing, but this is to persist all the failure logs in the test reports, right?  Like when `keep-going` is used.

Reply:
yeah, usually it deletes logs if it gets printed, this part is to make sure that the logs are always saved